recipes/lely-core/all/conandata.yml

```diff
@@ -0,0 +1,13 @@
+sources:
+  "2.3.1":
```

Comment:
You could make available only 2.3.2 if you want too. In case someone need 2.3.1, we can add in another PR. Less versions, less maintenance 😄

Reply:
https://github.com/conan-io/conan-center-index/blob/master/docs/adding_packages/sources_and_patches.md#adding-old-versions

Reply:
yeah, i also thought that made sense and was about to remove it, but didn't want to wait for a long for another review. I will remove it.